ops/utils.py

```diff
@@ -803,6 +803,38 @@ def applyIJ(f, arr, *args, **kwargs):
     # Reshape the resulting list of arrays to the determined output shape
     return np.array(arr_).reshape(output_shape)
 
+def applyIJ_parallel(f, arr, n_jobs=-2, backend='threading', tqdm=False, *args, **kwargs):
+    """
+    Decorator to apply a function that expects 2D input to the trailing two dimensions of an array,
+    parallelizing computation across 2D frames.
+
+    Parameters:
+        f (function): The function to be decorated and applied in parallel.
```

Comment:
I think in the future we should try to have better names for these function parameters. In this case, something like `applied_function` seems more descriptive of this parameter's use.

I don't think we need to fix this one before merging because right now this is a larger issue across the codebase, but we should keep it in mind going forward. This should make development for those with less context on our codebase (i.e., those making plugins) much easier.